Ginger/Ginger/ApplicationModelsLib/ModelParams Pages/ModelParamsPage.xaml.cs

```diff
@@ -35,11 +36,23 @@ namespace GingerWPF.ApplicationModelsLib.APIModelWizard
 {
     public partial class ModelParamsPage : Page
     {
-        ApplicationAPIModel mAAMB;
+        ApplicationModelBase mAAMB;
         public ObservableList<AppModelParameter> ParamsList = new ObservableList<AppModelParameter>();
         public ObservableList<GlobalAppModelParameter> APIGlobalParamList = new ObservableList<GlobalAppModelParameter>();
         string GridPlaceholderHeader = "Place Holder";
 
+        public event ModelParamEventHandler ParamDeleteEvent;
+        public delegate void ModelParamEventHandler(ParamEventArgs e);
+
+        public void OnParamDeleteEvent(List<AppModelParameter> paramsList)
+        {
+            ModelParamEventHandler handler = ParamDeleteEvent;
+            if (handler != null)
+            {
+                handler(new ParamEventArgs(paramsList));
+            }
+        }
+
         public ModelParamsPage(ApplicationAPIModel AAMB)
         {
```

Comment:
public ModelParamsPage(ApplicationAPIModel AAMB) - as disscused, ModelParamsPage supposed to get ApplicationModelBase to work for any model type, on Params deletion we need to check if the model type is API and then do all sync stuff

Reply:
Done